w3lib/url.py

```diff
@@ -342,12 +364,15 @@ def any_to_uri(uri_or_path):
     ).encode()
 )
 
-_ParseDataURIResult = namedtuple(
+ParseDataURIResult = namedtuple(
     "ParseDataURIResult", "media_type media_type_parameters data"
 )
+ParseDataURIResult.__doc__ = "The return value type of `w3lib.url.parse_data_uri`."
 
 
-def parse_data_uri(uri):
+# If we add the return type hint sphinx would error:
+#   w3lib/url.py:docstring of w3lib.url.parse_data_uri::py:class reference target not found: w3lib.url.ParseDataURIResult
+def parse_data_uri(uri: StrOrBytes):  # type: ignore
```

Comment:
What about adding `ParseDataURIResult` to the documentation, then (with https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html ). If we have documentation for a function, we should have documentation for its return value type.